mobile/src/main/java/org/openhab/habdroid/ui/activity/AbstractWebViewFragment.kt

```diff
@@ -91,8 +95,10 @@ abstract class AbstractWebViewFragment : Fragment(), ConnectionFactory.UpdateLis
                 .putExtra(MainActivity.EXTRA_SERVER_ID, context.getPrefs().getActiveServerId())
                 .setAction(shortcutAction)
 
-            webView?.url?.toHttpUrlOrNull()?.let {
-                intent.putExtra(MainActivity.EXTRA_SUBPAGE, it.toRelativeUrl())
+            launch {
```

Comment:
Instead of this and the below launch block, wouldn't it be better (and more consistent) to put the `pinShortcut` call in the JS interface into a launch block?

Reply:
Yes, makes sense.